Fix the GSL multifit 32-bit tests on x86 based on information provided at:
http://lists.gnu.org/archive/html/bug-gsl/2015-12/msg00018.html

This problem is known upstream. See http://savannah.gnu.org/bugs/?46593
which should hopefully provide the real fix.

--- gsl-2.1/multifit/test_wnlin.c.orig	2016-01-12 08:26:06.987977367 -0800
+++ gsl-2.1/multifit/test_wnlin.c	2016-01-12 12:29:34.092460907 -0800
@@ -4,7 +4,7 @@
 #define wnlin_NTRIES    1
 
 static double wnlin_x0[wnlin_P] = { 1.0, 0.0, 0.0 };
-static double wnlin_epsrel = 1.0e-9;
+static double wnlin_epsrel = 1.0e-8;
 
 static int wnlin_internal_weight = 1;
 
